airbyte-integrations/connectors/source-tplcentral/source_tplcentral/source.py

```diff
@@ -63,6 +75,11 @@ def refresh_access_token(self) -> Tuple[str, int]:
 
 class SourceTplcentral(AbstractSource):
     def _auth(self, config):
+        url_base = config.get("url_base")
+        if url_base.startswith("http://"):
```

Comment:
what happens if I enter "url_base" like "HtTp://" ?

Reply:
@arsenlosenko I suggest you'd better restrict this using regexp in a spec like [here](https://github.com/airbytehq/airbyte/pull/18607/files) and add a SAT test instead of a unit

Reply:
@davydov-d yep, it should be a better solution, I'll do that then, thanks

Reply:
@grubberr can you check again please, I've made the changes that were suggested